.github/workflows/tests.yml

```diff
@@ -307,6 +311,8 @@ jobs:
 
       - name: Run
         run: pytest --maxfail=5 -m "integration" test -k 'not tika'
+        env:
```

Comment:
I think it's better if we put this env vars on top of the workflow together with the other ones.

Reply:
Ok, would you please do that directly on the branch so we don't go back and forth about the exact place 🙏

Reply:
No back and forth needed, there are already other envs on top. 😁 
https://github.com/deepset-ai/haystack/blob/d2497d54e8e03b31d2fdfa3887ee515ec35b0940/.github/workflows/tests.yml#L22-L28